src/operator/src/expr_factory.rs

```diff
@@ -743,6 +744,20 @@ pub fn to_create_flow_task_expr(
     })
 }
 
+/// sanitize the flow name, remove possible quotes
+fn sanitize_flow_name(flow_name: ObjectName) -> Result<String> {
+    if flow_name.0.len() != 1 {
+        return InvalidFlowNameSnafu {
+            name: flow_name.to_string(),
+        }
+        .fail();
+    }
+    let ident = flow_name.0.first().context(InvalidFlowNameSnafu {
+        name: flow_name.to_string(),
+    })?;
+    Ok(ident.value.clone())
```

Comment:
Simplify the code and avoid a copy
```suggestion
    ensure!(
        flow_name.0.len() == 1,
        InvalidFlowNameSnafu {
            name: flow_name.to_string(),
        }
    );
    // safety: we've checked flow_name.0 has exactly one element.
    Ok(flow_name.0.swap_remove(0).value)
```